### PR TITLE
Updating after the pr went through

### PR DESCRIPTION
**Pull request by moxiegirl**
Updating doc for #12572

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

File: docs/man/docker-run.1.md [hunk 1]
```diff
@@ -397,17 +397,17 @@ read-only or read-write mode, respectively. By default, the volumes are mounted
...
 
-Two suffixes :z or :Z can be added to the volume mount. These suffixes tell
-Docker to relabel file objects on the shared volumes. The 'z' option tells
-Docker that the volume content will be shared between containers. Docker will
-label the content with a shared content label. Shared volumes labels allow all
-containers to read/write content. The 'Z' option tells Docker to label the
-content with a private unshared label. Private volumes can only be used by the
-current container.
+You can add either of two suffixes `:z` or `:Z` to the volume mount. These
+suffixes tell Docker to label file objects on the shared volumes. The `z`
```
Comment (thaJeztah) [inline docs/man/docker-run.1.md]: After having read the prior discussions on the PRs; the term "relabel" was used frequently. I'm not sure if it makes a difference in the description here, but I'd like to see that confirmed (from a technical perspective).

File: docs/sources/reference/commandline/cli.md [hunk 2]
```diff
@@ -2182,17 +2182,17 @@ the volumes are mounted in the same mode (read write or read only) as
...
 
-Two suffixes :z or :Z can be added to the volume mount. These suffixes tell
-Docker to relabel file objects on the shared volumes. The 'z' option tells
-Docker that the volume content will be shared between containers. Docker will
-label the content with a shared content label. Shared volumes labels allow all
-containers to read/write content. The 'Z' option tells Docker to label the
-content with a private unshared label. Private volumes can only be used by the
-current container.
+You can add either of two suffixes `:z` or `:Z` to the volume mount. These
+suffixes tell Docker to relabel file objects on the shared volumes. The `z`
```
Comment (thaJeztah) [inline docs/sources/reference/commandline/cli.md]: Oh, you *do* use "relabel" here.

**Comment by thaJeztah** [comment]
Two notes/question, but other than that LGTM

**Comment by moxiegirl** [comment]
@icecrime when you get a chance can you clarify what you mean by "relabel" . On the original, we say "relabel" is not allowed and then we say "to relabel" it was confusing.

**Comment by icecrime** [comment]
I don't want to say anything stupid, so I'm just gonna ping @rhatdan :innocent:

File: docs/man/docker-run.1.md (hunk 1 above)
Comment (SvenDowideit) [inline docs/man/docker-run.1.md]: yes, relabel is an important SELinux term that we should keep.

**Comment by SvenDowideit** [comment]
http://danwalsh.livejournal.com/38157.html :)

**Comment by rhatdan** [comment]
@SvenDowideit That guy tends to know what he is talking about...

**Comment by thaJeztah** [comment]
@rhatdan I heard he's good at coloring :pencil2: :pencil2: :pencil2: :pencil2: but only in a secure environment :smile:

**Comment by rhatdan** [comment]
Might be a new coloring book coming out soon, for the Red Hat Summit.

**Comment by moxiegirl** [comment]
LGTM author

**Comment by thaJeztah** [comment]
thanks @moxiegirl!

**Pull request merged**